.gitignore

```diff
@@ -2,6 +2,9 @@
 _build
 comment.md
 
+# Translation workflow
+src/beeware_docs_tools/docs
```

Comment:
This should only be relevant in this repo; otherwise it's symlinking into the `Path(__file__).parent` directory from the `build_pot_translation` file.